Neo4j.Driver/Neo4j.Driver.Reactive/Internal/InternalRxSession.cs

```diff
@@ -67,7 +67,7 @@ public IRxResult Run(string statement, object parameters, TransactionConfig txCo
         public IRxResult Run(Statement statement, TransactionConfig txConfig)
```

Comment:
Suggest to rename `IRxResult` to `IRxStatementResult` to be aligned with `IStatementResult` and `IStatementResultCursor`